website/src/theme/DocItem/Footer/contributors.module.scss

```diff
@@ -0,0 +1,63 @@
+.contributors {
+    margin-block: var(--ifm-spacing-vertical);
+}
+
+.wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0;
+    padding: 0;
+
+    li{
+        list-style: none;
+    }
+}
+
+.unleash{
+    position: relative;
+    display: block;
+}
+
+.unleash:before{
+    display: block;
+    position: absolute;
+    z-index: 2;
+    content: '';
+    top: -5px;
+    right: -5px;
+    width: 30px;
+    height: 30px;
+    background-image: url(/img/logo.svg);
+}
+
+.contributor {
+    margin: 5px 10px 5px 0;
+    display: flex;
+    gap: 10px;
```

Comment:
Why does the contributor have display flex and/or gap? There's only one item, right? And instead of setting the margin on the contributor, wouldn't it be easier to use the `gap` property on the `.wrapper`?

Doesn't really matter too much, though, just curious 💁🏼

Reply:
Good call, less code!